### PR TITLE
fix broken hyperlinks

### PR DESCRIPTION
**Pull request by eeyun**
A few broken links after rebuilding with the new homepage.

Also added a link to GitHub back to the footer.

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

**Comment by reset** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/202390691).

File: www/source/layouts/_nav.slim [hunk 1]
```diff
@@ -7,7 +7,7 @@
   = partial "layouts/message"
 
   a.main-nav--github
-    img src="/images/nav-github.svg" onerror="this.src='/images/nav-github.png'" alt="Find Us On Github"
+    img src="/images/nav-github.svg" onerror="this.src='/images/nav-github.png'" alt="Find Us On Github" href="https://github.com/habitat-sh"
```
Comment (Padgett) [inline www/source/layouts/_nav.slim]: href should be on line 9 with the `a` tag

Comment (eeyun) [inline www/source/layouts/_nav.slim]: awesome thank you @Padgett one moment

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/202390691).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/202390691).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**